compiler/src/iree/compiler/Preprocessing/Passes.cpp

```diff
@@ -90,9 +90,9 @@ void buildPreprocessingPassPipeline(
         preprocessingOptions.preprocessingPDLSpecFilename;
     passManager.addPass(
         Preprocessing::createApplyPDLPatterns(applyPDLPatternsOptions));
+    passManager.addPass(createCanonicalizerPass());
+    passManager.addPass(createCSEPass());
   }
-  passManager.addPass(createCanonicalizerPass());
-  passManager.addPass(createCSEPass());
```

Comment:
Can you elaborate on how this fixed the failing test? These passes shouldn't really be loading bearing (either including or excluding them)

Reply:
I added these passes to simplify lit tests that I am adding in the PR that looks at the iree-compile IR with the --compile-to=preprocessing stage. It is not load-bearing for the mlp-plugin work this PR is enabling. So why that crashes the tf test is something the engineers developing/maintaining that pipeline should investigate.

Reply:
oh and to answer how this fixes it, I am only running the clean up passes when I am running the pdl pass now, so it doesnt change the flow for other cases this way.

Reply:
Weird that the compilation flow is brittle there. Thanks for the fix.